dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -31,8 +31,17 @@ _wait_for_nodes_ready(){
     export KUBECONFIG=./kubeconfig
     ALL_READY_ITERATIONS=0
     ITERATIONS=0
-    # Node count is number of workers + 3 infra
-    NODES_COUNT=$(($2+3))
+    if [ "$3" == "rosa-spots=true" ]; then
+        if [ "$SPOT_POOL_READY" == "true" ]; then
+            # Node count is number of workers pool + 3 infra
+            NODES_COUNT=$(($2+3))
+        else
```

Comment:
Which condition needs `NODE_COUNT=$2`

Reply:
Good Catch. 
Before spot_pool_ready, I need to check whether the SPOT nodes are ready or not.